src/compiler/types.ts

```diff
@@ -1064,6 +1064,7 @@ module ts {
         getGlobalDiagnostics(): Diagnostic[];
         getSemanticDiagnostics(sourceFile?: SourceFile): Diagnostic[];
         getDeclarationDiagnostics(sourceFile?: SourceFile): Diagnostic[];
+        /* @internal */ getCompilerOptionsDiagnostics(): Diagnostic[];

```

Comment:
transpile is not an option. it is an entry point. the LS never uses transpile, nor does the commandline compiler.